python/ray/tests/ludwig/ludwig_test_utils.py

```diff
@@ -624,7 +625,8 @@ def train_with_backend(
             _, eval_preds, _ = model.evaluate(dataset=dataset)
             assert backend.df_engine.compute(eval_preds) is not None
 
-        return model.model.get_weights()
+        ret = True
```

Comment:
Why can't we continue to use `get_weights()` here?

Reply:
API has changed and that method is no longer present. As the return value wasn't being used, I just switched to this